content/_code-samples/secure-signing/py/sign-payment.py

```diff
@@ -0,0 +1,34 @@
+from xrpl.core import keypairs 
+
+# Define network client
+from xrpl.clients import JsonRpcClient
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Define signer address
+from xrpl.wallet import Wallet
+test_wallet_1 = Wallet(seed="shrPSF6vV3v3yoND9J3NeKks6M3xd")
+print(test_wallet_1.classic_address) # "raaFKKmgf6CRZttTVABeTcsqzRQ51bNR6Q"
+
+# Prepate the tranaaction
+from xrpl.ledger import get_latest_validated_ledger_sequence
+current_validated_ledger = get_latest_validated_ledger_sequence(client)
+min_ledger = current_validated_ledger + 1
+max_ledger = min_ledger + 20
+
+from xrpl.models.transactions import Payment
+# not working in testing
+# from xrpl.utils import xrp_to_drops
+my_payment = Payment(
+    account=test_wallet_1.classic_address,
+    amount="2200000",
+    destination="rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+    sequence=test_wallet_1.next_sequence_num,
```

Comment:
This won't always work depending on how/when you made the `Wallet()` instance and what the corresponding account has done on the ledger. If you, for example, paste in a seed from your password manager, it won't know what sequence number you're at until you explicitly look it up.

For purposes of this example code, we can just hard-code a sequence number. We should also hardcode a fee so that the transaction can be signed fully offline.